sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleObserverSdk.java

```diff
@@ -35,10 +37,11 @@
       boolean monotonic) {
     super(
         descriptor,
-        InstrumentValueType.LONG,
+        InstrumentValueType.DOUBLE,
         meterProviderSharedState,
         meterSharedState,
         monotonic);
+    collectLock = new ReentrantLock();
```

Comment:
since this is final and a simple assignment, it can be moved up to the declaration.

Reply:
done.